File: 58-length-of-last-word/58-length-of-last-word.c
int lengthOfLastWord(char* s) {

	size_t strSize = strlen(s);
	while (*(s + (strSize - 1)) == ' ') {
		*(s + (strSize - 1)) = '\0';
		strSize--;
	}

	size_t cnt = 0;
	while ((strSize > 0) && *(s + (strSize - 1)) != ' ') {
		*(s + (strSize - 1)) = '\0';
		strSize--;
		cnt++;
	}

	return cnt;
}

File: 58-length-of-last-word/58-length-of-last-word.py
class Solution:
    def lengthOfLastWord(self, s: str) -> int:
        
        while (s[-1] == " "):
            s = s[:-1]
        
        cnt = 0
        while ( len(s) > 0 and s[-1] != ' ' ):
            s = s[:-1]
            cnt += 1

        return cnt